<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListView</class>
 <widget class="QWidget" name="ListView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1030</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ListView</string>
  </property>
  <widget class="QGraphicsView" name="teethView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>701</width>
     <height>451</height>
    </rect>
   </property>
  </widget>
  <widget class="PatientTile" name="patientTile">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>461</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="AllergiesTile" name="allergiesTile">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>30</y>
     <width>351</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="statusEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="ControlPanel" name="controlPanel" native="true">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>350</y>
     <width>300</width>
     <height>334</height>
    </rect>
   </property>
  </widget>
  <widget class="SurfacePanel" name="surfacePanel" native="true">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>220</y>
     <width>291</width>
     <height>129</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="procedureTable">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>710</y>
     <width>701</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="procedureButton">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>710</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Добави манипулация</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PatientTile</class>
   <extends>QPushButton</extends>
   <header>tilebutton.h</header>
  </customwidget>
  <customwidget>
   <class>AllergiesTile</class>
   <extends>QPushButton</extends>
   <header>tilebutton.h</header>
  </customwidget>
  <customwidget>
   <class>ControlPanel</class>
   <extends>QWidget</extends>
   <header>ControlPanel/controlpanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SurfacePanel</class>
   <extends>QWidget</extends>
   <header>SurfacePanel/surfacepanel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
